components/hatch.py

```diff
@@ -1,63 +1,55 @@
-import math
-
 import wpilib
 
-from pyswervedrive.chassis import SwerveChassis
-
 
 class Hatch:
 
-    chassis: SwerveChassis
-
-    hatch_bottom_puncher: wpilib.Solenoid
-    hatch_left_puncher: wpilib.Solenoid
-    hatch_right_puncher: wpilib.Solenoid
-    hatch_wedge_piston: wpilib.DoubleSolenoid
+    hatch_fingers: wpilib.DoubleSolenoid
+    hatch_punchers: wpilib.Solenoid
+    hatch_enable_piston: wpilib.DoubleSolenoid
```

Comment:
Is there a particular reason why these are all prefixed with `hatch_` here? We know everything here is for the hatch mechanism...